testData/generate/GetterSetter1.hx

```diff
@@ -1,4 +1,5 @@
 class GetterSetter1 {
-  var foo:Int, bar:String;
+  var foo:Int;
+  var bar:String;
```

Comment:
syntax error fixed
confirmed it on try haxe web site that 
'var foo:Int, bar:String;' results in compile time error